web/satellite/src/App.vue

```diff
@@ -37,12 +37,26 @@ export default class App extends Vue {
 
     public mounted(): void {
         const meta = document.querySelector("meta[name='satellite-name']");
+        const segmentMeta = document.querySelector("meta[name='segment-io']");
         let satelliteName;
+        let segmentioID;
```

Comment:
segmentId ? usually we use camel case notation

Reply:
I didn't want this segment to be confused with the rest of the codebase